test/conditionals_test.dart

```diff
@@ -189,6 +189,36 @@ class SequentialModule extends Module {
   }
 }
 
+class SingleIfModule extends Module {
```

Comment:
Neither of these two classes are used?  Did you mean to add a test that uses them?

Reply:
Actually, it looks like the tests were there but got removed somehow in a merge?